build.sh

```diff
@@ -18,8 +18,8 @@ ARGS=$*
 # script, and that this script resides in the repo dir!
 REPODIR=$(cd $(dirname $0); pwd)
 
-VALIDARGS="clean librmm rmm -v -g -n -s --ptds -h"
-HELP="$0 [clean] [librmm] [rmm] [-v] [-g] [-n] [-s] [--ptds] [--cmake-args=\"<args>\"] [-h]
+VALIDARGS="clean librmm rmm -v -g -n -s --ptds --no-async -h"
```

Comment:
I think "--no-async" is a bit too broad of a name. There's a lot of async stuff in RMM...

Maybe `--no-cudamallocasync`.

Reply:
I like `--no-cudamallocasync` - very direct.